README.md

```diff
@@ -26,10 +26,51 @@ You then can invoke the CLI script using `vendor/bin/mysql-workbench-schema-expo
 
 Additionally to the [common options](https://github.com/mysql-workbench-schema-exporter/mysql-workbench-schema-exporter#configuring-mysql-workbench-schema-exporter) of mysql-workbench-schema-exporter these options are supported:
 
-### Sequelize Model
+### Sequelize 5
 
 Currently, no special options can be configured for Sequelize Model.
 
+### Sequelize 6
+
+  * `useSemicolons`
+
+    Whether or not to add semicolons to ends of lines (standard Eslint compliance).
+
+    Default is `true`.
+  
+  * `generateAssociationMethod`
+
+    Generate an association method to define associations between models.
+
+    You may instantiate your model this way :
+
+    ```javascript
+    const Sequelize = require('sequelize');
+
+    const sequelize = new Sequelize({...});
+    const MyModel1 = sequelize.import('./path/to/MyModel');
+    const MyModel2 = sequelize.import('./path/to/MyMode2');
+    ...
+
+    MyModel1.associate();
+    MyModel2.associate();
+    ...
+
+    ```
+    Default is `false`.
+
+  * `generateForeignKeysFields`
+
+    Whether or not to generate foreign keys fields and indexes. You could want to delegate it to association method (one could experiment relations creation order problems when it is present).
+
+    Default is `true`.
+
+  * `useTimestamps`
```

Comment:
`useTimestamp` property for table can be dropped, as it now can be supplied using dc74287494fd0788e369ad802cffbedeba42c560.